web/src/components/GuidedTour/homeStepList.tsx

```diff
@@ -1,5 +1,7 @@
 import {StepType} from '@reactour/tour';
```

Comment:
Do we need to keep this file?

Reply:
i rather have it in the if we ever need it